task/build-image-index/0.1/README.md

```diff
@@ -20,4 +20,5 @@ This takes existing Image Manifests and combines them in an Image Index.
 |IMAGE_URL|Image repository and tag where the built image was pushed|
 |IMAGES|List of all referenced image manifests|
 |IMAGE_REF|Image reference of the built image containing both the repository and the digest|
+|SBOM_BLOB_URL|Reference, including digest to the SBOM blob|
```

Comment:
Could you update this to match the description in the task?

You can do so by running `hack/generate-readme.sh task/build-image-index/0.1/build-image-index.yaml > task/build-image-index/0.1/README.md`